json_object.h

```diff
@@ -181,6 +181,12 @@ typedef enum json_type {
   json_type_string
 } json_type;
 
+/* json object int type, support extension*/
+typedef enum json_object_int_type {
```

Comment:
I was wondering if you were going to create a enum like this. :)
Please move this to json_object_private.h

Reply:
ok, done.